### ISSUE TITLE
Bug: Previous IP's always N/A

### ISSUE THREAD

**Issue by Jurrer** [qdm12/ddns-updater#606]
**TLDR**: *Describe your issue in a one liner here*
Even though updates.json has correct permissions, and is updated regularly, the Previous IP's section is always empty. 
Isn't the correct behavior to list Previous IP's according to `updates.json` contents?

1. Is this urgent: No
2. DNS provider(s) you use: not relevant
3. Program version:

    

    `Running version latest built on 2024-01-24T20:04:52.260Z (commit 7f0e858)`

4. What are you using to run the container: docker-compose
```
services:
  ddns-updater:
    image: qmcgaw/ddns-updater
    container_name: ddns-updater
    volumes:
      - ${HOMEPATH}/ddns-updater_cfg:/updater/data
      - ${BACKUPATH}/ddns-updater:/updater/backup
    environment:
      - TZ=${TZ}    
      - PERIOD=15m
      # Backup
      - BACKUP_PERIOD=168h # 0 to disable
      - BACKUP_DIRECTORY=/updater/backup
...
```
5. Extra information (optional)
![Screenshot from 2024-01-26 00-19-53](https://github.com/qdm12/ddns-updater/assets/72750942/c675d7cb-ec59-4cc5-b37d-766258f8549d)
Logs:

```log
...
2024-01-26T00:13:06+01:00 INFO Found 4 settings to update records
2024-01-26T00:13:06+01:00 INFO reading JSON config from file /updater/data/config.json
2024-01-26T00:13:06+01:00 INFO Reading history from database: domain *** host * ipv4
2024-01-26T00:13:06+01:00 INFO Reading history from database: domain *** host @ ipv4
2024-01-26T00:13:06+01:00 INFO Reading history from database: domain *** host www ipv4
2024-01-26T00:13:06+01:00 INFO Reading history from database: domain *** host ssh ipv4
...
```

Host OS:
Ubuntu Server

**Comment by Jurrer** [comment]
I didn't want to upload it, as it's mostly personally identifying information, [but here it is](https://pastebin.com/mDeh3FyG)

**Comment by qdm12** [comment]
You can remove it, that's a regression introduced recently, I have the same bug. Thanks for reporting it!

**Comment by qdm12** [comment]
Fixed in 74168ad4ab4852e019d6d89a0c3bce3e440d3e2b
This was a regression introduced in 7ed63a036ef310f24373802b89e2ac946844f2ca (my bad) two weeks ago. Thank you again for your help and time reporting all this 👍 !
